api/v1/controllers/aspects.js

```diff
@@ -29,6 +29,62 @@ const doPost = require('../helpers/verbs/doPost');
 const doPut = require('../helpers/verbs/doPut');
 const u = require('../helpers/verbs/utils');
 const httpStatus = require('../constants').httpStatus;
+const ZERO = 0;
+const ONE = 1;
+/**
+ * Given an array, return true if there
+ * are duplicates. False otherwise.
+ *
+ * @param {Array} tagsArr The input array
+ * @returns {Boolean} whether input array
+ * contains duplicates
+ */
+function checkDuplicates(tagsArr) {
```

Comment:
when a fn returns a boolean, it's good practice for the function name to reflect that, e.g. "hasDuplicates(...)" returns true if there are duplicates--it just makes it clearer what to expect